wurst/event/DamageEvent.wurst

```diff
@@ -0,0 +1,313 @@
+/** 
+    This package provides a light-weight damage detection system with priority-ordered listeners. 
+
+    **IMPORTANT** 
+        The system **must** be initialized using: initializeDamageEventSystem()
```

Comment:
does this have a particular reason?
What speaks against just doing it at init? (init blocks in external code is only executed when the package is imported directly or transitively in one of your packages)

Reply:
I had some issues with it not initializing when using classic init so I made the init manual at the time.  I will change it back to normal init and fill an issue if it happens again.